sdk/core/Azure.Core.Experimental/src/JsonData.cs

```diff
@@ -879,38 +423,11 @@ public IEnumerable<string> Properties
         /// Returns all the elements in this array.
         /// </summary>
         /// <remarks>If<see cref="Kind"/> is not<see cref="JsonValueKind.Array"/> this methods throws <see cref = "InvalidOperationException" />.</remarks>
-        public IEnumerable<JsonData> Items
+        internal IEnumerable<JsonData> Items
         {
             get => EnsureArray();
         }
 
-        /// <summary>
-        /// Writes the UTF-8 encoded string representation of this instance.
-        /// </summary>
-        /// <param name="stream">The stream to write to.</param>
-        /// <returns>The number of bytes written into the stream.</returns>
-        public long WriteTo(Stream stream)
-        {
-            using Utf8JsonWriter writer = new Utf8JsonWriter(stream);
-            WriteTo(writer);
-            writer.Flush();
-            return writer.BytesCommitted;
-        }
-
-        /// <summary>
-        /// Writes the UTF-8 encoded string representation of this instance.
-        /// </summary>
-        /// <param name="stream">The stream to write to.</param>
-        /// <param name="cancellationToken">A token to monitor for cancelation requests.</param>
-        /// <returns>The number of bytes written into the stream.</returns>
-        public async Task<long> WriteToAsync(Stream stream, CancellationToken cancellationToken)
```

Comment:
I removed this before understanding the requirement to get the JsonData as a stream.  We no longer want this as a public API on JsonData, but I will add this capability back in a different way.